src/main/java/com/hotels/plunger/TapUtil.java

```diff
@@ -48,4 +55,22 @@
     return null;
   }
 
+  /** Create a new {@link JobConf} which has all the values in the given {@link Configuration} */
+  static JobConf newJobConf(Configuration conf) {
```

Comment:
The two new methods don't ahve anything to do with Taps which is why I suggested putting them in a new JobUtils class. I'm OK with re-using this one for now and revisiting how methods are broken up across util classses if/when we need more of them. None of these classes is public so end users shouldn't be using them which means we can refactor as we see fit.